docker-compose/monitor/docker-compose-v2.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    # TODO: Need to use the local image here while making changes 
+    image: jaegertracing/jaeger
+    volumes:
+      - "./jaeger-ui.json:/etc/jaeger/jaeger-ui.json" # Do we need this for v2 ? Seems to be running without this.
+      - "./spm-v2.yml:/etc/jaeger/spm-v2.yml"
+    command: ["--config", "/etc/jaeger/spm-v2.yml"]
+    environment:
+      - METRICS_STORAGE_TYPE=prometheus
+      - PROMETHEUS_SERVER_URL=http://prometheus:9090
+      - PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR=${PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR:-true}
+      - PROMETHEUS_QUERY_NAMESPACE=${PROMETHEUS_QUERY_NAMESPACE:-}
+      - PROMETHEUS_QUERY_DURATION_UNIT=${PROMETHEUS_QUERY_DURATION_UNIT:-}
+      - PROMETHEUS_QUERY_NORMALIZE_CALLS=true
+      - PROMETHEUS_QUERY_NORMALIZE_DURATION=true
+    ports:
+      - "16686:16686"
+      - "14278:14278"
+  microsim:
+    networks:
+      - backend
+    image: yurishkuro/microsim:0.3.0
+    # TODO: Fix error - dial tcp [::1]:14278: connect: connection refused
+    command: "-j http://localhost:14278/api/traces -d 24h -s 500ms"
```

Comment:
Does microsim support exporting via OTLP? If not, might be a good thing to implement.